contracts/ethereum/test/ssv.test.ts

```diff
@@ -0,0 +1,134 @@
+import hre from 'hardhat'
```

Comment:
Reminder to myself to use these beautiful tests to understand what's going on myself.